pkg/apis/machine/v1beta1/machinehealthcheck_types.go

```diff
@@ -80,6 +80,17 @@ type MachineHealthCheckSpec struct {
 	// +kubebuilder:validation:Pattern="^([0-9]+(\\.[0-9]+)?(ns|us|µs|ms|s|m|h))+$"
 	// +kubebuilder:validation:Type:=string
 	NodeStartupTimeout metav1.Duration `json:"nodeStartupTimeout,omitempty"`
+
+	//Failed Machines older than this duration and are without a nodeRef or a providerID
+	//will be considered to have passed the time period allocated for manual fix and will be remediated.
+	//Expects an unsigned duration string of decimal numbers each with optional
+	//fraction and a unit suffix, eg "300ms", "1.5h" or "2h45m".
+	//Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
```

Comment:
Might be worth updating the node startup comment as well so that it's clear that won't apply to failed machines?
```suggestion
	// Failed Machines that are older than this value and are without a nodeRef or a providerID
	// will be considered to have passed the time period allocated for a manual fix and will be remediated.
	// Expects an unsigned duration string of decimal numbers each with optional
	// fraction and a unit suffix, eg "300ms", "1.5h" or "2h45m".
	// Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
```